blocks/library/gallery/block.scss

```diff
@@ -0,0 +1,39 @@
+
+.blocks-gallery {
```

Comment:
Aside and could be done separately, but we should probably drop this className in favour of the generated one `wp-block-gallery`